src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -943,8 +943,8 @@ private DetectProperties() {
             BooleanProperty.newBuilder("detect.component.location.analysis.enabled", false)
                     .setInfo("Component Location Analysis Enabled", DetectPropertyFromVersion.VERSION_8_11_0)
                     .setHelp(
-                            "If set to true, Detect will generate an output file detailing where dependencies are declared.",
-                            "All dependencies will be included when using Synopsys Detect in offline mode. Only policy violating dependencies will be included for Rapid and Stateless Scan modes.")
+                            "If set to true, Detect will save an output file named 'components-with-locations.json' in the Scan subdirectory detailing where in the project's source code OSS components are declared.",
+                            "All components will be included when using Synopsys Detect in offline mode. Only policy violating components will be included for Rapid and Stateless Scan modes.")
```

Comment:
nit: "policy violating" should probably be "policy-violating". (I realize this was here before but would be nice to fix).

Reply:
Why the hyphen? If it was here before, that may have been during this document's draft phase, is that when you saw it?